std/experimental/allocator/building_blocks/free_list.d

```diff
@@ -359,14 +359,27 @@ struct FreeList(ParentAllocator,
     }
 
     /**
-    Defined only if $(D ParentAllocator) defines $(D deallocateAll). If so,
-    forwards to it and resets the freelist.
+    If the parent allocator implements `deallocateAll` then forwards it otherwise
+    each element is freed with the parent's `deallocate`.
     */
-    static if (hasMember!(ParentAllocator, "deallocateAll"))
     bool deallocateAll()
     {
+        bool result = false;
```

Comment:
I can do it but since the code is basically a paste from ShareFreeList sould I also patch it ?